crates/wasmtime/src/runtime/component/types.rs

```diff
@@ -148,6 +148,7 @@ impl TypeChecker<'_> {
             (InterfaceType::Future(_), _)
             | (InterfaceType::Stream(_), _)
             | (InterfaceType::ErrorContext(_), _) => todo!(),
+            (InterfaceType::FixedSizeList(_), _) => todo!(),
```

Comment:
Since these are relatively targeted `todo!()` items mind filing an issue as a tracking issue and tagging these with `// FIXME(#NNNN)`?